lua/cmp/view/custom_entries_view.lua

```diff
@@ -31,7 +31,7 @@ custom_entries_view.new = function()
   self.entries_win:option('foldenable', false)
   self.entries_win:option('wrap', false)
   self.entries_win:option('scrolloff', 0)
-  self.entries_win:option('winhighlight', 'Normal:Pmenu,FloatBorder:Pmenu,CursorLine:PmenuSel,Search:None')
+  self.entries_win:option('winhighlight', 'Normal:Pmenu,CursorLine:PmenuSel,Search:None')
```

Comment:
I think this change will disallow to remove the border, right?

Reply:
No. This is only the highlighting of the border, not whether there is a border. I removed it because using `Pmenu` for the border made borders look odd. The default `FloatBorder` was good